### PR TITLE
feat: info popover on purses list

### PR DESCRIPTION
**Pull request by samsiegart**
fixes https://github.com/Agoric/agoric-sdk/issues/6507
Screen Shot 2022-11-02 at 9 58 52 PM

### CODE REVIEW

File: src/components/Card.jsx [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+  const isOpen = Boolean(anchorEl);
+
   return (
     <div className="Card">
       {header && (
         <div className="Header">
-          <h6>{header}</h6>
+          <h6>
+            {header}
+            {helptip && (
```
Comment (turadg) [inline src/components/Card.jsx:28]: so `helptip` is only observed if `header` is? worth some jsdoc on `Card`

File: src/components/Purses.jsx [hunk 2]
```diff
@@ -65,9 +65,24 @@ export const PursesWithoutContext = ({
     <Loading defaultMessage="Fetching purses..." />
   );
 
+  const helptip = (
+    <span>
+      The smart wallet only supports certain assets and IBC denoms. If you are
+      missing assets it may be because they are not supported.{' '}
+      <a
+        href="https://docs.inter.trade/inter-protocol-system-documentation/parity-stability-module/supported-assets"
```
Comment (turadg) [inline src/components/Purses.jsx:73]: this link is to a PSM resource. the smart wallet is outside PSM and even Inter Protocol. I think this Smart Wallet UI must not prefer any particular dapps or protocols.

Can this information be put somewhere neutral? E.g. on this repo?

File: src/components/tests/Dapp.test.jsx [hunk 3]
```diff
@@ -66,13 +66,13 @@ test('updates the petname when pressing enter on the textfield', () => {
 
   let textField = component.find(TextField);
   act(() =>
-    textField.props().onChange({ target: { value: 'Token Palace 2' } }),
+    textField.props().onChange({ target: { value: 'Token Palace 2' } })
```
Comment (turadg) [inline src/components/tests/Dapp.test.jsx:69]: the removal of the comma, seems like Prettier is lost. I'm working on fixing CI. meanwhile, please `prettier --write` over the PR

**Review by turadg — CHANGES_REQUESTED** [review]
Requesting fixes to formatting and resolution to the PSM/Inter coupling

File: src/components/tests/Dapp.test.jsx (hunk 3 above)
Comment (samsiegart) [reply src/components/tests/Dapp.test.jsx]: fixed

File: src/components/Card.jsx (hunk 1 above)
Comment (samsiegart) [reply src/components/Card.jsx:39]: done

**Review by turadg — APPROVED** [review]

**Pull request merged**